File: Seção 2/media.cs
using System;

namespace programa {

    class Program {

        static void Main(string[] args) {

            double x, y, media;

            Console.Write("Digite o primeiro numero: ");
            x = double.Parse(Console.ReadLine());
            Console.Write("Digite o segundo numero: ");
            y = double.Parse(Console.ReadLine());
            media = (x  + y) / 2.0;
            Console.WriteLine("Media = " + media);

        }

    }

}

File: Seção 2/media.c
#include <stdio.h>

int main() {

    double x, y, media;

    printf("Digite o primeiro numero: ");
    scanf("%lf", &x);
    printf("Digite o segundo numero: ");
    scanf("%lf", &y);
    media = (x + y) / 2.0;
    printf("Media = %.1f\n", media);
    return 0;

}